extensions/blocks/google-calendar/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { G, Rect, SVG, Path } from '@wordpress/components';
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import { extractAttributesFromIframe, URL_REGEX, IFRAME_REGEX } from './utils';
+import './editor.scss';
+
+export const name = 'google-calendar';
+export const title = __( 'Google Calendar', 'jetpack' );
+
+export const icon = (
+	<SVG height="23" viewBox="0 0 24 23" width="24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
In our other blocks we put the icon in a different file. Not a deal breaker, but would be good for consistency.

Reply:
done